README.adoc

```diff
@@ -5,3 +5,12 @@ NOTE: Extends link:https://github.com/jboss-container-images/jboss-openjdk-image
 # License
 
 See link:LICENSE[LICENSE] file.
+
+# TODO
+
+- [ ] Enable GitHub issue to allow people to give feedback
+- [ ] Add instructions how to recreate the image
```

Comment:
This is a good point, although we have more high priority issues now. But this will be added. As a README or a general instruction that applies to all our images.